apps/server/src/shared/domain/entity/file.entity.ts

```diff
@@ -28,6 +28,15 @@ export class Directory extends BaseFile {}
 
 @Entity({ collection: 'files', discriminatorValue: 'false' })
 export class File extends BaseFile {
+	@Property({ nullable: true })
```

Comment:
In the end we need all informations of the old model like s3 store, permissions and so on.